samples/torch/README.md

```diff
@@ -0,0 +1,47 @@
+# Torch demo
+
+Trains a handwritten digit classifier using the [Torch](http://torch.ch) C backend.
+Like other Torch clients, most prominently [PyTorch](http://pytorch.org),
+this example is built on top of the 
+[ATen C API](https://github.com/pytorch/pytorch/tree/master/aten),
+showing how a Torch client for Kotlin/Native could look like.
+
+## Installation
+
+To build [ATen (Torch for C)](https://github.com/pytorch/pytorch/tree/master/aten), 
+make sure you have Python 2.X and pyyaml installed:
+
+    # macOS: if you don't have pip
+    sudo easy_install pip
+    # Linux: if you don't have pip
+    apt-get -y install python-pip
```

Comment:
Why do we need Python at all here? K/N binary only depends on C code/headers.

Reply:
That is true, but the build process of ATen requires it, as described here: https://github.com/pytorch/pytorch/tree/master/aten